test/Sentry.Unity.Tests/SessionIntegrationTests.cs

```diff
@@ -23,5 +28,51 @@ public IEnumerator SessionIntegration_Init_SentryMonoBehaviourCreated()
             Assert.IsNotNull(sentryGameObject);
             Assert.IsNotNull(sentryMonoBehaviour);
         }
+
+        [UnityTest]
+        public IEnumerator SessionIntegration_SomethingElse()
+        {
+            yield return null;
+
+            var expectedRequestCount = 5;
+            var evt = new ManualResetEventSlim();
+            var requests = new List<string>();
+
+            using var _ = TestSentryUnity.Init(o =>
+            {
+                // o.AutoSessionTracking = true; We expect this to be true by default
+                o.AutoSessionTrackingInterval = TimeSpan.FromMilliseconds(10);
+                o.Debug = true;
+                o.DiagnosticLevel = SentryLevel.Info;
+            }, r =>
+            {
+                var request = r.Content.ReadAsStringAsync().GetAwaiter().GetResult();
+                Debug.Log(request);
+
+                requests.Add(request);
+
+                if (requests.Count >= expectedRequestCount)
+                {
+                    evt.Set();
+                }
+            });
+
+            var sentryGameObject = GameObject.Find("SentryMonoBehaviour");
+            var sentryMonoBehaviour = sentryGameObject.GetComponent<SentryMonoBehaviour>();
+
+            sentryMonoBehaviour.OnApplicationFocus(false);
+            yield return new WaitForSeconds(1f);
+            sentryMonoBehaviour.OnApplicationFocus(true);
+
+            if (!evt.Wait(TimeSpan.FromSeconds(3)))
+            {
+                Assert.Fail("Timeout");
+            }
+
+            // TODO: Use the envelope to parse the request and check for session
```

Comment:
I wonder if we can unblock this PR with a simple:

`requests.Any(r => r.Contains("\"status\":exited\")` of sorts. Where we just inspect for the part of the payload of the envelope we care.

Reply:
I'll give it a try!